test/test_v1.py

```diff
@@ -13,7 +13,38 @@ def test_v1_root():
     assert response.status_code == 200
     assert response.json() == {"version": 1}
 
-def test_breweries():
+
+def test_breweries_meta():
+    response = client.get("/v1/breweries/meta")
+    assert response.status_code == 200
+    assert response.json() == {"endpoint": "metadata"}
+
+
+def test_breweries_delete_id():
+    response = client.delete("/v1/breweries/1")
+    assert response.status_code == 501
+    assert response.json() == {"endpoint": "delete_brewery_by_id"}
+
+
+def test_post_breweries():
+    response = client.post("/v1/breweries")
+    assert response.status_code == 201
+    assert response.json() == {"endpoint": "post_breweries"}
```

Comment:
As mentioned at the endpoints, let's check for status 404. No other response check is needed, I don't think.

Reply:
done.